sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/aio/_azureappconfigurationproviderasync.py

```diff
@@ -30,6 +30,7 @@
 from azure.appconfiguration import (  # type: ignore # pylint:disable=no-name-in-module
```

Comment:
What is this file?

Reply:
Async version of the provider. Should be the same stuff but async. We do have a bit of code re-use were we can so they aren't exactly the same.

Reply:
Oh okay- it's not built from a build file or anything? You maintain both the async and sync separately? I just saw a lot of the same changes so I assumed it was a built file

Reply:
Yes we have to maintain both separately. I'd like to merge them a bit more when we get a chance to remove the extra common code. It just get's a bit tricky from where that common code is.